sdcclient/_client.py

```diff
@@ -270,31 +272,43 @@ def create_alert(self, name, description, severity, for_atleast_s, condition, se
             return [False, self.lasterr]
         j = res.json()
 
-        #
-        # Populate the alert information
-        #
-        alert_json = {
-            'alert' : {
-                'type' : 'MANUAL',
-                'name' : name,
-                'description' : description,
-                'enabled' : enabled,
-                'severity' : severity,
-                'timespan' : for_atleast_s * 1000000,
-                'condition' : condition,
-                'filter': user_filter
-            }
-        }
+        if alert_obj is None:
+            if None in (name, description, severity, for_atleast_s, condition):
+                return [False, 'Must specify a full Alert object or all parameters: name, description, severity, for_atleast_s, condition']
+            else:
+                #
+                # Populate the alert information
+                #
+                alert_json = {
+                    'alert' : {
+                        'type' : 'MANUAL',
+                        'name' : name,
+                        'description' : description,
```

Comment:
I would add something to the description which says this is restored from a script